2017/SEPT/SEPT17/CHEFSUM/CHEFSUM.c

```diff
@@ -0,0 +1,18 @@
+#include <stdio.h>
+int main()	{
+	int t;
+	scanf("%d", &t);
+	while (t--) {
+		int n, i, k = 100000, index;
+		scanf("%d", &n);
+		int arr[n];
+		for (i = 0; i < n; i++)	{
+			scanf("%d", &arr[i]);
+			if (arr[i] < k)	{
+				k = arr[i];
+				index = i;
+			}
+		}
+		printf("%d\n",index + 1);
+	}		
+}
```

Comment:
Line contains following spacing inconsistencies:
- No newline at EOF.

*Origin: SpaceConsistencyBear, Section: `c/cpp spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmptb9luzvn/2017/SEPT/SEPT17/CHEFSUM/CHEFSUM.c
+++ b/tmp/tmptb9luzvn/2017/SEPT/SEPT17/CHEFSUM/CHEFSUM.c
@@ -15,4 +15,4 @@
 		}
 		printf("%d\n",index + 1);
 	}		
-}+}
```